docs/capture-visualiser/capture-show-files.md

```diff
@@ -31,15 +31,19 @@ You will not be able to import files from newer versions of Capture - use the
 **"Export for Capture 2020"** function in the File menu of the full version of 
 Capture to save the file as a version which can then be imported.
 
-You can import fixtures from full Capture but you need to ensure that
-fixtures are correctly patched in Capture to match the patch on the
+When importing shows from full Capture you need to ensure that 
+all fixtures in the show have been correctly patched in Capture to match the patch on the
 console. There is no way to edit an externally created Capture patch on
 the console.
 
-Some fixture types and other Capture elements may not work when you
-import a show that has been created in full Capture. We recommend you
+You will need to turn off the **Auto Update** option in the Capture Settings window on Titan, otherwise Titan will move all the fixtures about according to layout settings in the Titan show.
+
+-  Some fixture types and other Capture elements may not work when you
+import a show that has been created in full Capture. You can
 use the [auto-patch function](../patching/patching-new-fixtures-or-dimmers.md#capture-visualiser-auto-patch) 
-to allow the console to patch the fixtures into Capture.
+to allow the console to patch fixtures into an imported Capture show, but you will then need to manually position them using the Titan controls.
```

Comment:
Not quite. It's talking about auto patch in the external vis I think. Basically when you patch with "console link" enabled it will come up with a popup in capture that will allow you to also place the fixtures in the external vis.

If you patch the fixtures in capture using this method you don't need to turn off auto update because the titan id will match up to the capture id properly already, so titan won't try and move them around. The case where you'll need to turn off auto update is when you've made a capture show totally independent of titan.

Reply:
I believe this section was referring to an imported Capture show, but early on in the Capture-Titan integration - there could be problems with fixtures which had imported from full capture being incompatible with the version of Capture on the console. So the advice was to use the auto-patch to replace the fixtures with the version installed on the console. I don't know if this is still the case.

You are right that this paragraph does not relate to the auto update issue, it is a separate issue with imported shows.

Reply:
Ah right my bad, I had read it as active fixture patch, but I now see it's talking about during import. My bad!